website/addons/box/client.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+from website.addons.base.exceptions import AddonError
+from website.addons.box.utils import refresh_creds_if_necessary
+from box import BoxClient
+
+
+def get_client(user):
+    #from website.addons.box.utils import refresh_creds_if_necessary
+    """Return a :class:`boxview.boxview.BoxView`, using a user's
+    access token.
+
+    :param User user: The user.
+    :raises: AddonError if user does not have the Box addon enabled.
+    """
+    user_settings = user.get_addon('box')
+    if not user_settings:
+        raise AddonError('User does not have the Box addon enabled.')
+    if refresh_creds_if_necessary(user_settings):
+        return BoxClient(user_settings.get_credentialsv2())
+    raise AddonError('Box credentials for this user have expired.')
+
+
+def get_client_from_user_settings(settings_obj):
```

Comment:
Shouldn't these methods be reversed? `get_client(user)` should call `get_client_from_user_settings`